File: template.py
# 
# https://codingcompetitions.withgoogle.com/

import sys
import math

T = int(input())
for t in range(T):

    A = [int(x) for x in input().split()]
    matrix_2d = [[0] * C for _ in range(R)]
    matrix_3d = [[[0] * N for _ in range(N)] for _ in range(N)]

    # print("{} {} {}".format(i + 1, j + 1, k + 1))
    # sys.stdout.flush()

    ans = 0

    print(f"Case #{t + 1}: {ans}")
    # print(f"Case #{t + 1}: {' '.join(map(str, ans))}")
    # print("bits", format(bits, f'#0{n+2}b'))

File: template.cpp
// 
// https://codingcompetitions.withgoogle.com/

#include <bits/stdc++.h>

typedef long long ll;

using namespace std;

int main() {
    int T;
    cin >> T;
    for (int t = 0; t < T; t++) {
        int M;
        cin >> M;
        vector< pair<long long, long long> > employees;
        vector< vector<int> > matrix_2d(26, vector<int>(26, LLONG_MAX));
        vector< vector< vector<ll> > > matrix_3d(C+1, vector< vector<ll> >(K+1, vector<ll>(2, LLONG_MAX)));
        for (int j = 0; j < L; j++) {
            cin >> num;
            cin >> level;
            employees.push_back(make_pair(level, num));
        }
        
        printf("Case #%d: %lld\n", t + 1, ans);
    }
}
